lib/src/Base/Optim/openturns/DlibFunctions.hxx

```diff
@@ -43,15 +43,16 @@ class DlibGradient
 {
 public:
   // Constructor
-  explicit DlibGradient(const Gradient & gradient);
+  DlibGradient(const Gradient & gradient, const Bool minimization = true);
```

Comment:
Why did you removed the explicit specifier?

Reply:
seems like a mistake